rules/rules_scoring.tex

```diff
@@ -1,11 +1,11 @@
 % !TEX TS-program = XeLaTeX
 The scoresheet is part of the \planetsheet.  The following items are scored:
 \begin{itemize}
-  \item \astronauts: one point for each \astronaut.
+  \item \astronauts: one point for each unused \astronaut.
```

Comment:
Not sure of that. Is it only unspent astronauts?

Reply:
The player should get points for the astronauts they deploy, even if unused.   It seems a player would not be rewarded only for astronauts they did not yet use, but rather ones they trained/used as well as those trained/on standby